webapp-client/public/index.html

```diff
@@ -1,26 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <link
-      href="%PUBLIC_URL%/fontawesome-free-5.13.1-web/css/all.css"
-      rel="stylesheet"
-    />
-    <link rel="icon" href="%PUBLIC_URL%/lobster_icon.jpg" />
-    <link rel="stylesheet" href="%PUBLIC_URL%/bulma.min.css" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000" />
-    <meta
-      name="description"
-      content="Web site for viewing and organizing saved travel clippings"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/lobster_icon_192.png" />
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <title>Lobster Ice Cream</title>
-  </head>
 
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-  </body>
+<head>
+  <meta charset="utf-8" />
+  <link rel="icon" href="%PUBLIC_URL%/lobster_icon.jpg" />
+  <link rel="stylesheet" href="%PUBLIC_URL%/bulma.min.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="theme-color" content="#000000" />
+  <meta name="description" content="Web site for viewing and organizing saved travel clippings" />
+  <link rel="apple-touch-icon" href="%PUBLIC_URL%/lobster_icon_192.png" />
+  <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+  <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+  <title>Lobster Ice Cream</title>
+</head>
+
+<body>
+  <noscript>You need to enable JavaScript to run this app.</noscript>
+  <div id="root"></div>
+</body>
+
```

Comment:
Do you and Anthony have the same lint setup? seems like you are toggling between indenting and new line here https://github.com/Sherry2001/lobster-ice/pull/30/files

Reply:
i didn't merge from master yet but i will, so the set up will be the same:)